scripts/bootstrap.py

```diff
@@ -15,8 +15,14 @@
 import subprocess
 import sys
 import tarfile
-import urllib2
-import urlparse
+
+
+try:  # python 3
+    from urllib.request import Request, urlopen
+    from urllib.parse import urlparse
+except ImportError:  # python 2
+    from urllib2 import Request, urlopen
+    from urlparse import urlparse
```

Comment:
Did you consider a compat module for things like this? They are rather repetitive.